### PR TITLE
feat: added a management command to start celery with auto-reload

### PR DESCRIPTION
**Pull request by inventhouse**
**Description:** Added a management command to start a celery worker process with Django's autoreload watchdog, so the worker will be restarted whenever the code changes; this should make local development easier and was suggested in [PR #7](https://github.com/edx/commerce-coordinator/pull/7)

**JIRA:**[[REV-2484] [ER-Impl] Set up Celery for auto-reload - JIRA](https://openedx.atlassian.net/browse/REV-2484)

**Testing instructions:**

- Set up and activate a virtualenv as described in [the README](https://github.com/edx/commerce-coordinator#one-time-setup) and [PR #7](https://github.com/edx/commerce-coordinator/pull/7)
- Map the redis port and launch it in devstack as described in [PR #7](https://github.com/edx/commerce-coordinator/pull/7) "Local Testing With Celery"
- Start commerce-coordinator with `./manage.py runserver localhost:8000 --settings=commerce_coordinator.settings.local` in one terminal
- Start the worker with `./manage.py celery` in another terminal
- In a third terminal, run `curl -s "http://localhost:8000/lms/test/" | jq '.'`
- Make a code change and save it; you should see celery restart automatically in the second terminal

**Author concerns:** I'm not sure any of our other projects run celery this way; perhaps this will change once we productionalize the coordinator into docker/devstack

### CODE REVIEW

File: commerce_coordinator/apps/core/management/commands/celery.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import subprocess
+
+from django.core.management.base import BaseCommand
+from django.utils import autoreload
+
+
+def restart_celery():
+    cmd = ('pkill', 'celery')
+    subprocess.call(cmd)
```
Comment (dianekaplan) [inline commerce_coordinator/apps/core/management/commands/celery.py:9]: Just to confirm, is there any chance another celery process could be running that we *don't* want to kill?

Comment (inventhouse) [reply commerce_coordinator/apps/core/management/commands/celery.py:9]: Good call! I  will try to craft a more targeted command.

Comment (inventhouse) [reply commerce_coordinator/apps/core/management/commands/celery.py]: So, turns out, funny story, that `pkill` command _wasn't actually cleaning up the old processes_ 🤦  The new version _does_ and is pretty specific to the celery instance being managed by the command.

**Review by dianekaplan — APPROVED** [review]
This looks great- thanks Ben! I guess there's one little 'helpful' commit message to throw a 'fix' on and we're good to go!

**Comment by inventhouse** [comment]
> I guess there's one little 'helpful' commit message to throw a 'fix' on and we're good to go!

Ugh, `doc:` should be `docs:` 😠  I'm gonna squash it away.

**Pull request merged**